### PR TITLE
DL-5850, cleaned hint and exclusive placeholders from Sequence labels

### PR DESCRIPTION
**Pull request by cmrhmrc**

### CODE REVIEW

File: test/core/models/StanzaHelper.scala [hunk 1]
```diff
@@ -47,18 +47,29 @@ trait StanzaHelper extends TestConstants {
...
+    )),
     Phrase(Vector("Apples", "Welsh: Apples")),
-    Phrase(Vector(
-      "Bananas [exclusive][hint:Selecting this checkbox will deselect the other checkboxes]",
-      "Welsh: Bananas [exclusive][hint:Welsh: Selecting this checkbox will deselect the other checkboxes]")),
+    Phrase(Vector("Bananas", "Welsh: Bananas")),
     Phrase(Vector("Oranges", "Welsh: Oranges"))
   )
 
+  // val multipleExclusiveSequencePhrases: Vector[Phrase] = phrases ++ Vector(
```
Comment (paulanderson1234) [inline test/core/models/StanzaHelper.scala]: Take it the above is a refactored version of these commented out lines ? If so do we need to retain  these?

Comment (cmrhmrc) [reply test/core/models/StanzaHelper.scala]: yes, will remove

**Comment by cmrhmrc** [comment]
Co-ticket on external-guidance also needs reviewing and merging, PR here
https://github.com/hmrc/external-guidance/pull/163

File: test/core/models/StanzaHelper.scala [hunk 2]
```diff
@@ -36,7 +36,7 @@ trait StanzaHelper extends TestConstants {
     Phrase(Vector("Text 8", "Welsh: Text 8"))
   )
 
-  val exclusiveSequencePhrases: Vector[Phrase] = phrases ++ Vector(
+  val rawExclusiveSequencePhrases: Vector[Phrase] = phrases ++ Vector(
```
Comment (paulanderson1234) [inline test/core/models/StanzaHelper.scala:39]: If this has been refactored to use rawExclusiveSequencePhrases ... then does this require to be used at lines 108, 111 - 113 ?

Comment (cmrhmrc) [reply test/core/models/StanzaHelper.scala:39]: no, different usage

Comment (paulanderson1234) [reply test/core/models/StanzaHelper.scala:39]: Discussed

File: test/core/models/StanzaHelper.scala (hunk 1 above)
Comment (paulanderson1234) [reply test/core/models/StanzaHelper.scala]: Verified as removed

**Pull request merged**